vendor/github.com/moby/buildkit/util/testutil/integration/run.go

```diff
@@ -257,7 +258,16 @@ func copyImagesLocal(t *testing.T, host string, images map[string]string) error
 				defer closer()
 			}
 		} else {
-			desc, provider, err = contentutil.ProviderFromRef(from)
+			dockerConfig := config.LoadDefaultConfigFile(os.Stderr)
```

Comment:
@crazy-max This brings in auth support for tests but still need to update CI side to pass the config into container as well.